tests/test_mananging_actor.py

```diff
@@ -8,20 +8,39 @@
 import logging
 from dataclasses import dataclass, replace
 from datetime import datetime, timedelta, timezone
-from typing import AsyncIterator, Iterator, cast
+from typing import AsyncIterator, Callable, Iterator, cast
+from unittest.mock import patch
 
 import async_solipsism
+import pytest
 import time_machine
 from frequenz.channels import Broadcast, Receiver, Sender
+from frequenz.client.dispatch import recurrence
 from frequenz.client.dispatch.recurrence import Frequency
+from frequenz.client.dispatch.test.client import FakeClient
 from frequenz.client.dispatch.test.generator import DispatchGenerator
 from frequenz.sdk.actor import Actor
 from pytest import fixture
 
-from frequenz.dispatch import ActorDispatcher, Dispatch, DispatchInfo
+from frequenz.dispatch import (
+    ActorDispatcher,
+    Dispatch,
+    Dispatcher,
+    DispatchInfo,
+    MergeByIdentity,
+    MergeByType,
+    MergeByTypeTarget,
+    MergeStrategy,
+)
 from frequenz.dispatch._bg_service import DispatchScheduler
 
 
+@fixture
+def generator() -> DispatchGenerator:
+    """Return a dispatch generator."""
+    return DispatchGenerator()
```

Comment:
For the future, we can register new fixtures to `pytest` automatically in the client by making it a [`pytest` plugin](https://docs.pytest.org/en/6.2.x/writing_plugins.html?highlight=setuptools#writing-plugins), then client users could just magically use `dispatch_generator` as a fixture without creating one or importing it, like we can do with `mocker` and other fixtures.